daml_dit_ddit/subcommand_run.py

```diff
@@ -48,10 +49,10 @@ def subcommand_main(
     with open(RUNTIME_DIT_META_NAME, 'w') as runtime_meta_file:
         runtime_meta_file.write(package_meta_yaml(dabl_meta))
 
-    if os.path.isfile(INTEGRATION_ARG_FILE):
-        LOG.info(f'Argument file found: {INTEGRATION_ARG_FILE}')
+    if os.path.isfile(args_file):
+        LOG.info(f'Argument file found: {args_file}')
     else:
-        LOG.info(f'Argument file not found.')
+        LOG.info(f'Argument file not found: {args_file}')
         subcommand_genargs(integration_type_id)
```

Comment:
I love the idea, particularly when testing multiple configurations of the same integration.

However, I do think the `genargs` command should be extended with the same option (which should be passed in here in l56.) That way, the overall workflow is left unchanged when you pass in this new option.

Reply:
Ah, good idea, I'll add that